doc/source/serve/api/index.md

```diff
@@ -66,6 +67,7 @@ This is fixed by added custom filename mappings in `source/conf.py` (look for "a
    serve.delete
    serve.start
    serve.shutdown
+   serve.status
```

Comment:
Since there are different sets of code owners, I feel it might be better to just split this into it's own PR, but no biggie

Reply:
Good point, will keep in mind for future PRs!